idea/jvm-debugger/jvm-debugger-test/test/org/jetbrains/kotlin/idea/debugger/test/util/KotlinOutputChecker.kt

```diff
@@ -75,7 +77,20 @@ internal class KotlinOutputChecker(
             LOG.error("Test file created ${outFile.path}\n**************** Don't forget to put it into VCS! *******************")
         } else {
             val originalText = FileUtilRt.loadFile(outFile, CharsetToolkit.UTF8)
-            val expected = StringUtilRt.convertLineSeparators(originalText)
+            val ignoreDirectives = mutableListOf<TargetBackend>()
+            val expected = StringUtilRt.convertLineSeparators(originalText).split("\n").filter {
+                if (it.trim().startsWith("// IGNORE_BACKEND: ")) {
```

Comment:
Consider `InTextDirectivesUtils.findListWithPrefixes` usage

Reply:
Thank you! I ended up simplifying most of this away by just using the utility function to check if the test is ignored for the current backend.

Reply:
Cool! I forgot about it